File: app/src/main/java/com/burial/test/MainActivity2.java
package com.burial.test;

import android.app.Activity;
import android.os.Bundle;
import android.os.PersistableBundle;
import android.support.annotation.Nullable;
import android.util.Log;
import com.yan.burial.method.timer.BurialTimer;

public class MainActivity2 extends Activity {

  @Override
  protected void onCreate(@Nullable Bundle savedInstanceState) {
    super.onCreate(savedInstanceState);
    setContentView(R.layout.activity_main);
    BurialTimer.getTimer().setListener(
        (ignore, className, methodName, des, cost) -> Log.e("BurialTimer",
            className + "  " + methodName + "  " + des + "  " + des));
  }

  @Override public void onCreate(@Nullable Bundle savedInstanceState,
      @Nullable PersistableBundle persistentState) {
    super.onCreate(savedInstanceState, persistentState);
  }

  @Override protected void onPostCreate(@Nullable Bundle savedInstanceState) {
    super.onPostCreate(savedInstanceState);
  }

  @Override public void onPostCreate(@Nullable Bundle savedInstanceState,
      @Nullable PersistableBundle persistentState) {
    super.onPostCreate(savedInstanceState, persistentState);
  }

  @Override protected void onStart() {
    super.onStart();
  }

  @Override protected void onPostResume() {
    super.onPostResume();
  }

  @Override protected void onResume() {
    super.onResume();
  }
}

File: app/src/main/java/com/burial/test/MainActivity2.kt
package com.burial.test

import android.app.Activity
import android.os.Bundle
import android.os.PersistableBundle
import android.util.Log
import com.burial.test.R.layout

class MainActivity2 : Activity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(layout.activity_main)
        test3()
    }

    override fun onCreate(
            savedInstanceState: Bundle?,
            persistentState: PersistableBundle?
    ) {
        super.onCreate(savedInstanceState, persistentState)
    }

    override fun onPostCreate(savedInstanceState: Bundle?) {
        super.onPostCreate(savedInstanceState)
    }

    override fun onPostCreate(
            savedInstanceState: Bundle?,
            persistentState: PersistableBundle?
    ) {
        super.onPostCreate(savedInstanceState, persistentState)
    }

    override fun onStart() {
        super.onStart()
        Presenter().forClass()
    }

    override fun onPostResume() {
        super.onPostResume()
    }

    override fun onResume() {
        super.onResume()
    }

    fun test() {
        try {
            Thread.sleep(100)
        } catch (e: InterruptedException) {
            e.printStackTrace()
        }
        Log.i("test", "testtesttesttesttesttesttesttesttesttest")
    }

    fun test2() {
        val persenter = Presenter()
    }

    fun test3() {
        test()
        test2()
    }
}